index.js

```diff
@@ -1,16 +1,23 @@
 /***** Deliverable 1 *****/
-const header = document.querySelector("#header")
-console.log("Here's your header:", header)
+const header = document.querySelector(`#header`)
+console.log("Here's your header:")
 
 
 /***** Deliverable 2 *****/
-
+const newHeader = document.querySelector('#header')
+console.log(newHeader.style.color = 'blue')
 
 
 /***** Deliverable 3 *****/
-// console.log('This is what the traveler object looks like: ', traveler)
-
-
+console.log('This is what the traveler object looks like: ', traveler)
+const updatedTraveler = document.querySelector('div#profile img')
+updatedTraveler.src = "pics/raffy.jpg"
+updatedTraveler.alt = "Raffy"
+
+const updatedName = document.querySelector('div#profile h2')
+updatedName.textContent = "Chris"
+const updatedName2 = document.querySelector('div#profile em')
+updatedName2.textContent = "Kim"
 /***** Deliverable 4 *****/
 
 
```

Comment:
**here's some hints w/ deliverable 4**

You need to create [an `li`](https://www.w3schools.com/tags/tag_li.asp) for _each_ animal sighting post and the parent should be the `animalsUl`. You have several options: 
Change the `innerHTML` of the `animalsUl`
```js
const animalsUl = document.querySelector('ul#animals');
traveler.animalSightings.forEach( function(sighting) {
  animalsUl.innerHTML += `<li data-id="${sighting.id}">
  <p>${sighting.description}</p>
  <img src="${sighting.photo}" alt="${sighting.species}"/>
  <a href="${sighting.link}" target="_blank">Here's a video about the ${sighting.species} species!</a>
  </li>`
});
```